cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-ui.adoc

```diff
@@ -0,0 +1,290 @@
+----
++
+.Example output
++
+[source,terminal]
+----
+I: Creating User role
+I: Creating ocm user role using 'arn:aws:iam::000000000000:user/rosa-user'
+I: Created role 'ManagedOpenShift-User-rosa-user-Role' with ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-User-rosa-user-Role'
+I: Linking User role
+I: Successfully linked role ARN 'arn:aws:iam::000000000000:role/ManagedOpenShift-User-rosa-user-Role' with account '1rbOQez0z5j1YolInhcXY000000'
+----
++
+[NOTE]
+====
+As before, you can define `--mode manual` if you'd prefer to execute the AWS CLI commands yourself. The CLI outputs the AWS commands and the relevant JSON files are created in the current directory. Make sure to link the role.
+====
+
+. Click *Ok*.
+
+== Confirming successful account association
+
+. You will be brought back to the original window. You should now see your AWS account in the drop down menu. If you see your account, account association was successful.
+
+. Select the account.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-associate3.png[]
+
+. You will see the account role ARNs populated below.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-account-roles.png[]
+
+. Click *Next*.
+
+== Creating the cluster
+
+. For the purposes of this workshop make the following selections:
++
+.Cluster settings
++
+* Cluster name: **<pick a name\>**
+* Version: **<select latest version\>**
+* Region: **<select desired region\>**
+* Availability: **Single zone**
+* Enable user workload monitoring: **leave checked**
+* Enable additional etcd encryption: **leave unchecked**
+* Encrypt persistent volumes with customer keys: **leave unchecked**
+
+. Click *Next*.
+
+. Leave the default settings on for the machine pool:
++
+.Default machine pool settings
++
+* Compute node instance type: **m5.xlarge - 4 vCPU 16 GiB RAM**
+* Enable autoscaling: **unchecked**
+* Compute node count: **2**
+* Leave node labels blank
+
+. Click *Next*.
+
+=== Networking
+
+. Leave all the default values for configuration.
+
+. Click *Next*.
+
+. Leave all the default values for CIDR ranges.
+
+. Click *Next*.
+
+=== Cluster roles and policies
+For this workshop, leave *Auto* selected. It will make the cluster deployment process simpler and quicker.
+
+[NOTE]
+====
+If you selected a *Basic OCM role* earlier, you can only use manual mode. You must manually create the operator roles and OIDC provider. See the "Basic OCM role" section below after you have completed the "Cluster updates" section and started cluster creation.
+====
+
+=== Cluster updates
+* Leave all the options at default in this section.
+
+=== Reviewing and creating your cluster
+. Review the content for the cluster configuration.
+. Click *Create cluster*.
+
+=== Monitoring the installation progress
+* Stay on the current page to monitor the installation progress. It should take about 40 minutes.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-cluster-create.png[]
```

Comment:
Installation progress step screenshot seems outdated. Please correct it.
Reference screenshot with latest flow
![image](https://github.com/openshift/openshift-docs/assets/93653094/b8aaf2af-6792-4565-976c-86986a2272e1)